lib/upload.js

```diff
@@ -245,29 +264,37 @@ class BaseUpload {
    * @api private
    */
   _startParallelUpload () {
-    const totalSize = this._size = this._source.size
+    this._size = this._source.size
+    const totalSize = this._size
```

Comment:
I wanted to fix some of the lints in this file so that I could easily spot new warnings introduced by my code, which ended up being really useful in development.

Most of the lints were fixed in this commit, it won't take long to review.

https://github.com/SkynetLabs/tus-js-client/pull/6/commits/8d54e716b5c89da5d2e5c8a79dd1a8a9beb7cbb9